cpu/stm32l1/include/cpu_conf.h

```diff
@@ -72,6 +72,13 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+ * @brief   Flash page configuration
+ * @{
+ */
+#define FLASHPAGE_SIZE      (256U)
+#define FLASHPAGE_NUMOF     (2048U)
```

Comment:
This is off: I am pretty sure the flash has not the same size for all members of the `stm32l1` family -> these values have to be guarded with the actual `CPU_MODEL_xx` values.

Reply:
True, will fix.